tests/test_mask.py

```diff
@@ -140,10 +140,10 @@ def testArrays(self):
     def testInitializeMasks(self):
         val = 0x1234
         msk = afwImage.Mask(lsst.geom.ExtentI(10, 10), val)
-        self.assertEqual(msk.get(0, 0), val)
+        self.assertEqual(msk[0, 0, afwImage.PARENT], val)
```

Comment:
see comment in `test_background.py`. There's no need to include the origin since `PARENT` is the default value.